doc/tests/empty.rst

```diff
@@ -1,11 +1,20 @@
 ``empty``
 =========
 
-``empty`` checks if a variable is empty:
+``empty`` checks if a variable is
+ - an empty string
+ - an empty array
+ - exactly ``false``
+ - exactly ``null``
+ - an instance of Countable_ and has a count_ of zero
```

Comment:
An empty array is already mentioned, so I guess just remove "an instance of Countable_ and has a count_ of zero" and add "an empty hash"?